core/src/main/java/org/neo4j/ogm/autoindex/AutoIndexManager.java

```diff
@@ -263,11 +265,16 @@ private static Set<AutoIndex> initialiseAutoIndex(MetaData metaData) {
 
             if (needsToBeIndexed(classInfo)) {
 
+                AnnotationInfo relationshipEntityAnnotation = classInfo.annotationsInfo().get(RelationshipEntity.class);
+                boolean isRelationShip = relationshipEntityAnnotation != null;
```

Comment:
```suggestion
                boolean isRelationship = relationshipEntityAnnotation != null;
```
minor thing, but please :)